test/runtests.jl

```diff
@@ -123,7 +123,7 @@ Random.seed!(0)
 
   @testset "Enzyme" begin
     Pkg.add(["CUDA", "cuDNN"])
-    import Enzyme
+    using Enzyme
```

Comment:
CI says "WARNING: using Enzyme.gradient in module Main conflicts with an existing identifier."

```suggestion
    import Enzyme
```